force-app/main/default/classes/StorageChargeCheck.cls

```diff
@@ -0,0 +1,62 @@
+public with sharing class StorageChargeCheck implements Database.Batchable<sObject>
+{
```

Comment:
Let's follow Java coding conventions and put { (braces) on the same line with method/class declaration

Reply:
It makes code hard to read. I don't like Java convention